andela/pulse-mobile-app
# [#156629488] Add the Side Drawer Interface

#### What does this PR do?
Create user interface for the Side Drawer.

#### Description of Task to be completed?
UI implementation of the Side Drawer according to mockup.

#### How should this be manually tested?
- run react-native _run-android_ and _react-native run-ios_ for android and ios respectively. 
- run yarn test. This should run the tests

#### What are the relevant pivotal tracker stories?
[#156629488](https://www.pivotaltracker.com/story/show/156629488)

#### Screenshots (if appropriate)

#### Check List

#### Questions:
